test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -113,13 +116,15 @@ def _construct_tensor_for_quantization_test(
     return tensor
 
 
-def _parameterized_class_attrs_and_values():
+def _parameterized_class_attrs_and_values(max_opset_version: int):
     attrs = ("opset_version", "is_script", "keep_initializers_as_inputs")
     input_values = []
     input_values.extend(itertools.product((7, 8), (True, False), (True,)))
     # Valid opset versions are defined in torch/onnx/_constants.py.
-    # Versions are intentionally set statically, to not be affected by elsewhere changes.
-    input_values.extend(itertools.product(range(9, 17), (True, False), (True, False)))
+    # Versions are intentionally set statically, to not be affected by changes elsewhere.
+    input_values.extend(
+        itertools.product(range(9, max_opset_version + 1), (True, False), (True, False))
```

Comment:
There are talks of changing the minimum opset version too. I'd recommend parametrizing this along with max version

Reply:
I suggest doing that after we know changing the minimum is something we decide to do. What do you think?

Reply:
I'd do now to prevent another PR touching the same files that this PR is already changing.

Generally speaking, code refactoring (although good) makes the tail of files/source-code lines change history longer, making `git blame` or `git bisect` process and therefore the debug process to take longer. If we can add the same value to the source code in fewer PRs, I'd prefer it (assuming the PRs are making only related changes, not a crazy blob of code change lol)

Reply:
ok! Done